3_javascript/javascript1/week1/Startup-name.js

```diff
@@ -0,0 +1,7 @@
+const firstWords = ["Easy", "Awesome", "Corporate", "Super", "Good"];
+        const secondWords = ["Corporation", "City", "Working", "Power", "World"];
+        const randomNumber = Math.floor(Math.random() * 5);     
+        let startupName = firstWords[randomNumber] + " " + secondWords[randomNumber];
+        console.log(startupName);
+        let length = startupName.length
```

Comment:
your length variable contains also an empty space between words, which is not actually a character , to have precise number of character you can use this, for example : l`et length = startupName.length - 1`